projects/admin/src/app/migration/conversion/record/detail-view/migration-data/migration-data.component.html

```diff
@@ -0,0 +1,43 @@
+<!--
+  RERO ILS UI
+  Copyright (C) 2020-2024 RERO
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published by
+  the Free Software Foundation, version 3 of the License.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+@if (record()) {
```

Comment:
Why you make `record()` ? With Syntax `@Input()` the correct syntax is `record`

Reply:
It is a signal: https://github.com/rero/rero-ils-ui/pull/1202/files/3e4ca78710b8ed13dc2d83e2824403ac50549688#diff-973db89ae02a60930e2dceaa16ad378d48dd1984f724c63cc12ec771ec990751R38

Reply:
My apologies, I think I read the wrong component.ts